Packs/AWS_SystemManager/Integrations/AWSSystemManager/AWSSystemManager.py

```diff
@@ -0,0 +1,692 @@
+        kwargs["NextToken"] = next_token
+
+    response = ssm_client.get_inventory(**kwargs)
+    command_results = []
+
+    if response_next_token := response.get("NextToken"):
+        command_results.append(
+            next_token_command_result(response_next_token, "InventoryNextToken"),
+        )
+
+    entities = response.get("Entities", [])
+    command_results.append(
+        CommandResults(
+            outputs_prefix="AWS.SSM.Inventory",
+            outputs=entities,
+            outputs_key_field="Id",
+            readable_output=tableToMarkdown(
+                name="AWS SSM Inventory",
+                t=_parse_inventory_entities(entities),
+            ),
+        ),
+    )
+    return command_results
+
+
+def list_inventory_entry_command(ssm_client: "SSMClient", args: dict[str, Any]) -> list[CommandResults]:
+    """Lists inventory entries for a specific instance and type name using the provided SSM client and arguments.
+
+    Args:
+    ----
+        ssm_client: AWS SSM client object for making API requests.
+        args (dict): Command arguments containing filters and parameters.
+            - instance_id (str): The ID of the instance.
+            - type_name (str): The type name of the inventory.
+            - limit (int, optional): Maximum number of entries to retrieve. Defaults to 50.
+            - next_token (str, optional): Token to retrieve the next set of entries.
+
+    Returns:
+    -------
+        list[CommandResults]: A list of CommandResults containing the inventory entries information.
+            and the next token if exists.
+
+    Raises:
+    ------
+        DemistoException: If an invalid instance ID is provided.
+    """
+    def _parse_inventory_entries(entries: list[dict[str, Any]]) -> list[dict[str, Any]]:
+        """Parses a list of inventory entries and returns a list of dictionaries containing relevant information.
+
+        Args:
+        ----
+            entries (list[dict]): A list of inventory entries to parse.
+
+        Returns:
+        -------
+            list[dict]: A list of dictionaries containing relevant information for each inventory entry.
+        """
+        return [
+            {
+                "Instance Id": entry.get("InstanceId"),
+                "Computer Name": entry.get("ComputerName"),
+                "Platform Type": entry.get("PlatformType"),
+                "Platform Name": entry.get("PlatformName"),
+                "Agent version": entry.get("AgentVersion"),
+                "IP address": entry.get("IpAddress"),
+                "Resource Type": entry.get("ResourceType"),
+            }
+            for entry in entries
+        ]
+
+    validate_args(args)
+
+    kwargs = {
+        "InstanceId": args["instance_id"],
+        "TypeName": args["type_name"],
+        "MaxResults": arg_to_number(args.get("limit", 50)) or 50,
+    }
+    kwargs.update({"NextToken": next_token}) if (next_token := args.get("next_token")) else None
```

Comment:
This is a confusing use of a ternary operator. a simple `if` would be clearer for future maintainers
```suggestion
    if next_token := args.get("next_token"): 
        kwargs["NextToken"] = next_token
```
Thanks for the inspiration for https://github.com/astral-sh/ruff/issues/6850 😅